src/components/afm/tests/utils/dummyXirrFixture.ts

```diff
@@ -0,0 +1,138 @@
+// (C) 2007-2019 GoodData Corporation
+import { Execution, AFM } from "@gooddata/typings";
+
+export const executionRequest = {
```

Comment:
Maybe type executionRequest instead of just aggregation

Reply:
Typed, thanks